pkg/plugin/plugin.go

```diff
@@ -16,6 +16,33 @@ type Plugin struct{}
 
 var _ api.Plugin = &Plugin{}
 
+func (p *Plugin) MergeConfig(cs, oldCs *acsapi.ContainerService) {
+	if oldCs == nil {
+		return
+	}
+
+	// generated config should be copied as is
```

Comment:
In the future this will have to have logic too. Question if it will be here, or in other places. In example image version changes, new kubeconfig, etc.